pulsar-broker/src/test/java/org/apache/pulsar/broker/admin/AdminApi2Test.java

```diff
@@ -924,6 +925,23 @@ public void testUpdatePartitionedTopicProperties() throws Exception {
         Assert.assertEquals(properties.size(), 2);
         Assert.assertEquals(properties.get("key1"), "value11");
         Assert.assertEquals(properties.get("key2"), "value2");
+
+        // create topic without properties
+        admin.topics().createPartitionedTopic(topicNameTwo, 2);
+        properties = admin.topics().getProperties(topicNameTwo);
+        Assert.assertNull(properties);
+        // remove key of properties on this topic
+        admin.topics().removeProperties(topicNameTwo, "key1");
+        Assert.assertNull(properties);
```

Comment:
No need this line I think

Reply:
It is to verify the case that `p.properties` is `null`
https://github.com/apache/pulsar/pull/17337/files#r958309585